controller/internal/routes/transit_router_api_model.go

```diff
@@ -101,7 +101,8 @@ func MapTransitRouterToApiEntity(appEnv *env.AppEnv, context *response.RequestCo
 		IsOnline:    appEnv.GetDbProvider().GetControllers().Routers.IsConnected(txRouter.Id),
 	}
 
-	if !txRouter.IsVerified {
+
+	if !txRouter.IsBase && !txRouter.IsVerified {
```

Comment:
`IsBase` allows external handler and API level functions to know how to render list display items.